Maquetado/views/index.html

```diff
@@ -15,12 +15,12 @@
     <div class='backButtonContainer'>
       <a href='http://127.0.0.1:5500/Maquetado/views/bookList.html' class='backButton'> Atrás </a>
       <a href='http://127.0.0.1:5500/Maquetado/views/bookList.html'>
-        <img src='../resources/img/back-arrow.png'class='backArrowImage' alt='Back Arrow Image'/> 
+        <img src='../resources/img/back-arrow.png' class='backArrowImage' alt='Back Arrow Image' />
       </a>
     </div>
```

Comment:
Sorry because this is not a change from this PR, but I'm almost sure that this can be simplified in a simple <a> with a pseudo element in css